src/ert/storage/local_storage.py

```diff
@@ -346,7 +346,12 @@ def _migrate(self, ignore_migration_check: bool) -> None:
                 experiment_id.migrate(self.path)
                 update_params.migrate(self.path)
                 empty_summary.migrate(self.path)
+                observations.migrate(self.path)
```

Comment:
Should not be in v4? Though I think this needs a rebase on main

Reply:
wrt. `_measured_data` & transpose, I think it is done, the "dataframe" is stored in "long" format.
The current format of the data, as it is stored is:
```
                          OBS  STD          0  ...         47        48         49
name     key_index                             ...                                
POLY_OBS [0, 0]      2.145705  0.6   3.721637  ...   2.457555  0.862469   2.625992
         [2, 0]      8.769220  1.4   6.419814  ...   5.933843  1.304883   4.650068
         [4, 0]     12.388015  3.0  12.796416  ...  11.487246  2.535165   8.349348
         [6, 0]     25.600465  5.4  22.851445  ...  19.117763  4.553314  13.723831
         [8, 0]     42.352048  8.6  36.584901  ...  28.825395  7.359332  20.773518
[5 rows x 52 columns]

```
internally it is stored as a numpy array with some getters etc

(in https://github.com/equinor/ert/pull/7495)

Reply:
wrt migrations, I think it should be written in the same way as the refactored migrations are 
(in https://github.com/equinor/ert/pull/7563)